WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -200,7 +215,10 @@ - (XCElementSnapshot *)fb_parentCellSnapshot
                                                   @(XCUIElementTypeIcon),
                                                   ];
   if (self.elementType != XCUIElementTypeCell && self.elementType != XCUIElementTypeIcon) {
-    targetCellSnapshot = [self.lastSnapshot fb_parentMatchingOneOfTypes:acceptableElementTypes];
```

Comment:
I would consider creating two methods 
`fb_parentMatchingOneOfTypes:` and `fb_parentMatchingOneOfTypes:filter:`. First could just call second with filter this filter or nil (then second could check for filter).